mllib/src/test/scala/org/apache/spark/ml/optim/WeightedLeastSquaresSuite.scala

```diff
@@ -531,7 +534,8 @@ class WeightedLeastSquaresSuite extends SparkFunSuite with MLlibTestSparkContext
          standardization <- Seq(false, true)) {
       for (solver <- WeightedLeastSquares.supportedSolvers) {
         val wls = new WeightedLeastSquares(fitIntercept, regParam, elasticNetParam = 0.0,
-          standardizeFeatures = standardization, standardizeLabel = true, solverType = solver)
+          standardizeFeatures = standardization, standardizeLabel = true, solverType = solver,
```

Comment:
The updated tol is needed to allow it to pass - and matches the tol in the R code that was used to generate 'correct' answers, so seems OK. Same with maxIter. I made a similar change throughout this test suite for consistency.